source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -27,14 +27,10 @@ bool isCorsPreflightRequest(const Http::RequestHeaderMap& headers) {
          !headers.getInlineValue(access_control_request_method_handle.handle()).empty();
 }
 
+// The prefix used in the response code detail sent from jwt authn filter.
+constexpr char kRcDetailJwtAuthnPrefix[] = "jwt_authn_access_denied";
```

Comment:
nit: make this absl::string_view to avoid any strlen lookup.

Reply:
Done. Thanks!